api/test/robot_tests/plugin_manager/plugin_manager.robot

```diff
@@ -19,7 +19,7 @@ GET /plugin-manager with no plugins
 
 POST /plugin-manager?source={{pluginSource}}
     ${source}=    Set Variable
-    ...    https://github.com/massalabs/thyra-plugin-hello-world/releases/download/0.0.6/thyra-plugin-hello-world_${OS}-${ARCH}.zip
+    ...    https://github.com/massalabs/thyra-plugin-hello-world/releases/download/0.0.7/thyra-plugin-hello-world_${OS}-${ARCH}.zip
```

Comment:
Can you also use a constant ?

Reply:
I put it in this file api/test/robot_tests/plugin_manager/variables.resource